projects/uitdatabank/docs/search-api/filters/location.md

```diff
@@ -1,14 +1,284 @@
-Coming soon.
-
-<!-- 
-  @todo 
-  Explain the following URL parameters and when to use them / what for:
-    regions[]
-    coordinates
-    distance
-    bounds
-    sort[distance]
-    postalCode
-    addressCountry (mention default filter on this and link to that guide)
-    locationId
--->
+# Filtering on location
+
+It is possible to filter the results from Search API based on the geographical information. There are many different options, which we outline in detail in this guide.
+
+## Filtering on country
+
+You can filter results from Search API based on the country the event / place / organizer is in.
+
+> By default, the search API will only return results that are in Belgium. In order to retrieve results outside Belgium you'll need to disable the default filter of the addressCountry URL parameter. You can reset this default as described in the [default filters guide](../filters/default-filters.md).
+
+The `addressCountry` is available as both an URL parameter and as an advanced query field:
+* URL parameter: `addressCountry`
+* Advanced query parameter: `address.{language}.addressCountry`
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+The value for this field must always be an [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code.
+
+**Examples**
+
+Because a document can have address translations you can search by a specific language, like in the example below:
+
+```
+GET /places/?q=address.nl.addressCountry:NL&addressCountry=*
+```
+
+Use a wildcard instead of specifying a language:
+
+```
+GET /places/?q=address.\*.addressCountry:NL&addressCountry=*
+```
+
+In the queries above, we disabled the addressCountry default filter using a URL parameter and applied a filter for the Netherlands through the advanced query parameter (`address.nl.AddressCountry`). However, if the goal is to restrict results solely to locations within the Netherlands, it’s simpler to use just the URL parameter:
+
+```
+GET /places/?addressCountry=NL
+```
+
+## Filtering on regions
+
+publiq has a list of pre-indexed geographical shapes that represent the administrative state of Belgium. The geographical coördinates of events and places are then matched with pre-indexed geographical shapes for:
+* Provinces
+* Regions
+* Municipalities
+* Submunicipalities
```

Comment:
Ik denk dat dit niet aan elkaar geschreven wordt, samentrekkingen in het engels zijn zeldzaam. Wikipedia scheidt ze met een scheidingstreep (https://en.wikipedia.org/wiki/Category:Sub-municipalities_of_Belgium), mijn editor zegt dan weer om ze gewoon apart te schrijven.

Reply:
Fixed in commit `fa52894`